TOMP-API.yaml

```diff
@@ -619,8 +620,34 @@ paths:
           $ref: "#/components/responses/401Unauthorized"
         "404":
           $ref: "#/components/responses/404NotFound"
+    post:
```

Comment:
Nope. As described in the blueprint, there is an /legs/{id}/events with 'ASSIGN_ASSET'.